### DISCUSSION TITLE
Offline cheatsheet in lazyvim, just like nvchad.

### DISCUSSION THREAD

**Discussion by harshv5094** [LazyVim/LazyVim#4252] [Ideas]
Before permanently shifting to lazyvim, I was using NvChad for my config. I love it's inbuilt cheatsheet feature, is it possible to introduce this feature in lazyvim as an extra feature. It's very useful since I don't have to open browser side by side to look shortcut

**Comment by dpetka2001** [comment]
You can use `:Telescope keymaps` to see the keymaps that are set.

**Comment by qoodboop** [comment]
Hello with Lazyvim you could use  [ leader + sk ]  in order to display all shortcuts

**Reply by harshv5094** [reply]
I just checked out, its very polish, I just need to type description now
